south-african-journal-of-animal-science.csl

```diff
@@ -39,7 +39,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name delimiter=", " and="text" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="first"/>
+      <name delimiter=", " and="symbol" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="always"/>
```

Comment:
(you can update this pull request by using the edit icon at https://github.com/Yusufras/styles/blob/patch-16/south-african-journal-of-animal-science.csl)